impala/assets/dashboards/impala_overview.json

```diff
@@ -1561,8 +1714,7 @@
     }
   ],
   "layout_type": "ordered",
-  "is_read_only": false,
   "notify_list": [],
   "reflow_type": "fixed",
-  "id": "87p-bm2-b7x"
```

Comment:
I'm not sure if this affects anything, but just to be safe can we leave this the same too?

Reply:
It does not, but let's keep the old one 🙂